spec/controllers/user_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+  # before(:all) do
+  #   User.create(id: 1, name: 'emmy', email: 'emmy@gmail.com', password: '123456', password_confirmation: '123456')
+  #   User.create(id: 2, name: 'emma', email: 'emma@gmail.com', password: '123456', password_confirmation: '123456')
+  #   @friendship = Friendship.create(requestor_id: 1, requestee_id: 2)
+  # end
+
```

Comment:
It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
See this [post](https://stackoverflow.com/questions/13130859/inline-comments-in-ruby/13130900#:~:text=No%2C%20Ruby%20does%20not%20have,comment%20out%20the%20one%20row), about why comments are discouraged.

Reply:
Thanks @acushakoncept for the insights.